public/app/plugins/panel/heatmap/heatmap_tooltip.ts

```diff
@@ -85,8 +87,10 @@ export class HeatmapTooltip {
 
     let tooltipTimeFormat = 'YYYY-MM-DD HH:mm:ss';
     let time = this.dashboard.formatDate(xData.x, tooltipTimeFormat);
-    let decimals = this.panel.tooltipDecimals || 5;
-    let valueFormatter = this.valueFormatter(decimals);
+
+    let decimals = this.panel.tooltipDecimals || this.panelCtrl.decimals;
+    let scaledDecimals = this.panel.tooltipDecimals ? decimals - 2 : this.panelCtrl.scaledDecimals;
```

Comment:
That's a bit tricky. We have `decimals` options both for Y axis and tooltip. Graph panel uses this algorithm:
```js
          // decimal override
          if (_.isNumber(panel.decimals)) {
            series.updateLegendValues(formater, panel.decimals, null);
          } else {
            // auto decimals
            // legend and tooltip gets one more decimal precision
            // than graph legend ticks
            var tickDecimals = (axis.tickDecimals || -1) + 1;
            series.updateLegendValues(formater, tickDecimals, axis.scaledDecimals + 2);
          }
```

So, maybe we should use the same?